src/form/GeometryField.tsx

```diff
@@ -0,0 +1,141 @@
+import React, {useState, useEffect} from 'react';
+import {FloatInput} from './FloatInput';
+import {
+  AssetLocationValue
+} from "../types/locationFormTypes";
+import formStyles from "../styles/Forms.module.css";
+
+ 
+
+interface MyProps {
+  title: string,
+  name: string,
+  value: AssetLocationValue,
+  validated: boolean,
+  valueChanged: (value: AssetLocationValue) => void,
+  clearInput?: (e: any) => void,
+  errorMessage?: string | false,
+  placeholder?: string,
+  handleEnter?: (e: any) => void,
+  onFocus?: (e: React.ChangeEvent<HTMLInputElement>) => void,
+  onBlur?: () => void,
+  triedToSubmit?: boolean,
+  readOnly?: boolean
+};
+
+export const GeometryField: React.FC<MyProps> = (props) => {  
+  const {
+    title,
+    name,
+    placeholder,
+    value,
+    validated,
+    valueChanged,
+    onFocus,
+    onBlur,
+    handleEnter,
+    clearInput,
+    errorMessage,
+    triedToSubmit,
+    readOnly
+  } = props;
+
+  const [lat, setLat] = useState(NaN);
+  const [lng, setLng] = useState(NaN);
```

Comment:
missing type definition for lat and lng

Reply:
and any reason to use NaN instead of null as initial value here?

Reply:
Using "null / undefined / {} / other empty values" is always a difficult thing for me.
I thought to use NaN here because it indicates that the type is a number.
Anyway, I have no standard way of doing this.
If you have a reason to rather use null then NaN I am also fine.